dist/elemica-suggest.js

```diff
@@ -27,6 +31,7 @@ elemicaSuggest 0.9.2-SNAPSHOT
         ALT = 18;
         OS_LEFT = 91;
         OS_RIGHT = 92;
+        NON_PRINTALBE_KEYS = [SHIFT, CTRL, ALT, OS_LEFT, OS_RIGHT, TAB, LEFT_ARROW, RIGHT_ARROW];
```

Comment:
`PRINTALBE` -> `PRINTABLE` ? :wink:

Reply:
I think you can blame natives, I have just added `LEFT_ARROW` and `RIGHT_ARROW` ;-)

Reply:
Fixed